scripts/config.properties.sample

```diff
@@ -0,0 +1,6 @@
+# Example config file for updateDatabase.sh
```

Comment:
Could the file name be changed, maybe to sample_config.properites, so .properties is the extension?